tools.html.erb

```diff
@@ -39,21 +38,42 @@
       e.preventDefault();
     })
 } );
+
+$(document).on("click", '.show-more a', function(e) {
```

Comment:
As this script is getting bigger you might want to consider moving this to a js file

Reply:
Yes, totally, I want to have everything separated in html/css/js files.